scripts/ci/kubernetes/kube/templates/configmaps.template.yaml

```diff
@@ -54,6 +54,9 @@ data:
     statsd_port = 8125
     statsd_prefix = airflow
 
+    # optional comma-separated prefix whitelist (e.g: scheduler,executor,dagrun)
+    statsd_whitelist =
```

Comment:
You don't need to add this here.

Reply:
But isn't a good idea to keep the configmap.yaml templates aligned with the default `.cfg` template?

Reply:
It's been what we have done in the past (so probably can leave it for now) but it is in no way required. This configmap template provides the airflow.cfg for the pod, and airflow's config looks for the first match in

1. airflow.cfg in the AIRFLOW_HOME
2. default_airflow.cfg from the dist.

Since we have updated the default we don't also need to update the configmap.

But for consistency now it's okay.

Reply:
Ok, I'm going to remove it for reducing the PR scope.